### ISSUE TITLE
no message and job and process id when a job is started

### ISSUE THREAD

**Issue by sabinem** [fish-shell/fish-shell#5062]
`fish` gives you nothing, when you start a job in the background:
-  no process id, no job number, no message.
```
> sleep 100 &
```

`bash` in contrast gives you both: job and process id. Is that as it should be or should I report this as an issue?

```
> sleep 100 &
[3] 35051
```

I think this information would be very much worth reporting also in `fish`, so that a user can better identify the jobs he started.  

(fish, version 2.7.1-1246-geca4273f-dirty)

**Comment by faho** [comment]
I don't really feel like this information is all that useful - if you want it, you can run `jobs`. But I also don't tend to use background processes all that much (I'd rather open another shell).

Any other opinions on this?

**Comment by sabinem** [comment]
@faho : I can understand that you rather open another shell window, but I think that is because fish is not very strong on this. I feel job control is an overall neglected topic in fish, but fish does not communicate this clear enough to the user.

fish has user friendliness as one of its design principles, which I like a lot. It is userfriendly in many ways.

But regarding job control there is no user friendliness: Newcomers like me are surprised to find out that commands written in fish as fish functions cannot be started as background jobs. 

fish does everything with commands, but not all commands are equal regarding job control. This is not documented as far as I know. As a user I would expect clear rules and guidelines on this. 

People coming from bash are used, that they can send just about anything to the background. 

Even though I feel it is okay that fish focusses on interactive shells, I still think, that this shoud be then communicated to the user in the documentation and the messages he receives.

**Comment by faho** [comment]
>I feel job control is an overall neglected topic in fish

That is correct. It's quite complicated technically, and not really all that flashy, so it doesn't tend to attract much attention.

Let's keep the bit about functions etc in #238.

-----

One thing I'd like to know regarding your use of backgrounded jobs: Do you interact with them again? If we printed this message (or something with more... accessible wording), would you use that information?

Because to me it seems like you'd often background jobs that do some work and then quit. Stuff like our use of it - `fish_update_completions &`. In that case you don't want or need the job id or process group, and if you decided you wanted it, you could execute `jobs` to look it up.

If, however, I am wrong (which is quite possible!), and the majority of backgrounded jobs are ones you interact with, by sending signals and whatnot, then obviously the message is a win.

Or maybe it's just about showing _that_ it's still executing?

**Comment by sabinem** [comment]
@faho : 

> Or maybe it's just about showing that it's still executing?

Exactly! I wont to see what is happening with my commands: 
- clear input
- clear result

I would wish for:
- a message, when I cann't put something into the background, rather then a reaction: sort of a sanity check of the command entered: so I know I better use another terminal window for this and won't even start it here
- a message with the job id if it was started so I can relate it to the message I receive when my job was done

I just want to know what is going on: If I speak to a human and he doesn't answer, I feel ignored. The same goes for speaking to a shell. If their is nothing: no response I feel ignored and I am not sure, whether it got my command in the way I intended it.
